composer.json

```
@@ -46,7 +46,7 @@
         "aws/aws-sdk-php": "^3.38",
         "guzzlehttp/guzzle": "^7.2",
         "guzzlehttp/psr7": "^1.0",
-        "lcobucci/jwt": "^3.3|^4.0",
+        "lcobucci/jwt": "^4.1 || ^5.3",
```

The 4.1 branch requires PHP `^7.4 || ^8.0` so it'll never get installed.

```suggestion
        "lcobucci/jwt": "^5.3",
```